src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py

```diff
@@ -26,6 +27,10 @@ def __init__(self, **kwargs):
         #this will break all in one test
         self.reqMgr2 = ReqMgr(kwargs.get("reqmgr2_endpoint", None))
         self.previous_state = {}
+        localurl = kwargs.get('logdb_url', '')
+        identifier = kwargs.get('hostName', '')
```

Comment:
do you prefer the  hostName over log_reporter? :smile: 
I can change that but for GQ and reqmgr is not really a host name. 
https://github.com/dmwm/deployment/pull/176/files#diff-0d7567a7c6e08d29a72ba8e06afacc5fR63